module/zfs/zio_compress.c

```diff
@@ -20,9 +20,9 @@
  */
 
 /*
- * Copyright (c) 2009, Sun Microsystems Inc. All rights reserved.
- * Copyright (c) 2013, Saso Kiselkov. All rights reserved.
- * Copyright (c) 2013, 2018, Delphix. All rights reserved.
+ * Copyright (c) 2009, Sun Microsystems Inc.
+ * Copyright (c) 2013, Saso Kiselkov.
```

Comment:
let's drop the periods maybe? rest looks fine :-)

Reply:
Dropped those after person names, kept them after `inc.`